person.rb

```diff
@@ -1,16 +1,17 @@
-require './nameable'
+require './decorators/nameable'
 require 'securerandom'
 
 class Person < Nameable
-  attr_reader :id
-  attr_accessor :name, :age
+  attr_reader :id, :rentals
+  attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ] Based on the requirement, kindly create a setter method for the rentals attribute. You can use this shorthand method 🖱️ 💻:

```ruby
attr_accessor :rentals
```